### PR TITLE
feat: add manual command

### PR DESCRIPTION
**Pull request by xZreai**
## Description

adds manual command to support category

## Test Results
image

## Discord Username

Cleon#5802

### CODE REVIEW

File: src/commands/support/manual.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const MANUAL_URL = 'https://cdn.discordapp.com/attachments/785976111875751956/927657411764969612/unknown.png';
+
+export const manual: CommandDefinition = {
+    name: 'manual',
+    description: 'Displays image about manual waypoints',
+    category: CommandCategory.A32NX,
```
Comment (benw202) [inline src/commands/support/manual.ts:10]:
```suggestion
    category: CommandCategory.SUPPORT,
```

You have added this to the support category (at least in folder structure), so we need to define this here too.

File: src/commands/support/manual.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    executor: (msg) => msg.channel.send(makeEmbed({ 
+        title: 'FlyByWire Support | Manual Leg Diagram',
+        image: { url: MANUAL_URL } })),
```
Comment (benw202) [inline src/commands/support/manual.ts:13]:
```suggestion
        image: { url: MANUAL_URL }, 
    })),
```

Just some formatting

**Review by benw202 — CHANGES_REQUESTED** [review]

**Comment by xZreai** [comment]
oop didn't mean to request review

File: src/commands/support/manualleg.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const MANUAL_URL = 'https://cdn.discordapp.com/attachments/785976111875751956/927657411764969612/unknown.png';
+
+export const manualleg: CommandDefinition = {
+    name: ['manualleg','vm'],
+    description: 'Displays image about manual waypoints',
+    category: CommandCategory.A32NX,
```
Comment (benw202) [inline src/commands/support/manualleg.ts:10]:
```suggestion
    category: CommandCategory.SUPPORT,
```

**Review by benw202 — CHANGES_REQUESTED** [review]

**Comment by Earthsam12** [comment]
Can you use the [new image](https://docs.flybywiresim.com/pilots-corner/advanced-guides/flight-planning/disco/?h=manual#illustrations-for-manual-legs) by mav?

**Comment by xZreai** [comment]
> Can you use the [new image](https://docs.flybywiresim.com/pilots-corner/advanced-guides/flight-planning/disco/?h=manual#illustrations-for-manual-legs) by mav? And maybe link to that page in the footer too?

Yep done

**Comment by xZreai** [comment]
Can I get any ideas for what the text description for this command would be?

File: src/commands/support/manualleg.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const MANUAL_URL = 'https://cdn.discordapp.com/attachments/752801628347957248/929071827752480839/mcdu-discontinuity-manual-conceptual.png';
+
```
Comment (Earthsam12) [inline src/commands/support/manualleg.ts:6]:
```suggestion
const DISCON_DOCS_URL = 'https://docs.flybywiresim.com/pilots-corner/advanced-guides/flight-planning/disco/?h=manual#discontinuities-after-manual-leg';

```

File: src/commands/support/manualleg.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+export const manualleg: CommandDefinition = {
+    name: ['manualleg','vm'],
```
Comment (Earthsam12) [inline src/commands/support/manualleg.ts:8]:
```suggestion
    name: ['manualleg', 'vm'],
```

File: src/commands/support/manualleg.ts [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    description: 'Displays image and links to docs about manual waypoints',
+    category: CommandCategory.SUPPORT,
+    executor: (msg) => msg.channel.send(makeEmbed({
```
Comment (Earthsam12) [inline src/commands/support/manualleg.ts:11]:
```suggestion
    executor: (msg) => msg.channel.send(makeEmbed({
```

File: src/commands/support/manualleg.ts [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+export const manualleg: CommandDefinition = {
+    name: ['manualleg','vm'],
+    description: 'Displays image and links to docs about manual waypoints',
+    category: CommandCategory.SUPPORT,
+    executor: (msg) => msg.channel.send(makeEmbed({ 
+        title: 'FlyByWire Support | Manual Leg',
+        image: { url: MANUAL_URL },
+        description : makeLines ([
+            'Please see our [documentation](https://docs.flybywiresim.com/pilots-corner/advanced-guides/flight-planning/disco/?h=manual#discontinuities-after-manual-leg) for information on how to deal with a manual leg in your flight plan.'
```
Comment (Earthsam12) [inline src/commands/support/manualleg.ts:15]:
```suggestion
            `Please see our [documentation](${DISCON_DOCS_URL}) for information on how to deal with a manual leg in your flight plan.`
```

File: src/commands/support/manualleg.ts [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+        ]),
+     })),
```
Comment (Earthsam12) [inline src/commands/support/manualleg.ts:17]:
```suggestion
    })),
```

**Review by Earthsam12 — CHANGES_REQUESTED** [review]
remove some out of place whitespace and also shorten the line so that vsc doesn't yell at you

File: src/commands/support/manualleg.ts [hunk 9]
```diff
@@ -0,0 +1,19 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const MANUAL_URL = 'https://cdn.discordapp.com/attachments/752801628347957248/929071827752480839/mcdu-discontinuity-manual-conceptual.png';
```
Comment (benw202) [inline src/commands/support/manualleg.ts:5]:
```suggestion
const MANUAL_LEG_IMAGE_URL = 'https://cdn.discordapp.com/attachments/752801628347957248/929071827752480839/mcdu-discontinuity-manual-conceptual.png';
```

Clears up constant naming. 'MANUAL' could be mistaken for a traditional manual etc when searching the repo.

File: src/commands/support/manualleg.ts [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+const DISCON_DOCS_URL = 'https://docs.flybywiresim.com/pilots-corner/advanced-guides/flight-planning/disco/?h=manual#discontinuities-after-manual-leg'
+
+export const manualleg: CommandDefinition = {
+    name: ['manualleg','vm'],
+    description: 'Displays image and links to docs about manual waypoints',
+    category: CommandCategory.SUPPORT,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire Support | Manual Leg',
+        image: { url: MANUAL_URL },
```
Comment (benw202) [inline src/commands/support/manualleg.ts:14]:
```suggestion
        image: { url: MANUAL_LEG_IMAGE_URL },
```

File: src/commands/support/manualleg.ts [hunk 11]
```diff
@@ -0,0 +1,19 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const MANUAL_URL = 'https://cdn.discordapp.com/attachments/752801628347957248/929071827752480839/mcdu-discontinuity-manual-conceptual.png';
+const DISCON_DOCS_URL = 'https://docs.flybywiresim.com/pilots-corner/advanced-guides/flight-planning/disco/?h=manual#discontinuities-after-manual-leg'
+
+export const manualleg: CommandDefinition = {
+    name: ['manualleg','vm'],
```
Comment (benw202) [inline src/commands/support/manualleg.ts:9]:
```suggestion
    name: ['manualleg', 'vm'],
```

This was suggested before, but not done.

**Review by benw202 — CHANGES_REQUESTED** [review]

**Review by benw202 — APPROVED** [review]

**Pull request merged**